docs/index.rst

```diff
@@ -115,19 +115,19 @@ Create from timestamps (ints or floats, or strings that convert to a float):
     >>> arrow.get('1367900664.152325')
     <Arrow [2013-05-07T04:24:24.152325+00:00]>
 
-Use a naive or timezone-aware datetime, or flexibly specify a time zone:
+Use a naive or timezone-aware datetime, or flexibly specify a timezone:
 
 .. code-block:: python
 
     >>> arrow.get(datetime.utcnow())
     <Arrow [2013-05-07T04:24:24.152325+00:00]>
 
-    >>> arrow.get(datetime.now(), 'US/Pacific')
-    <Arrow [2013-05-06T21:24:32.736373-07:00]>
+    >>> arrow.get(datetime(2013, 5, 5), 'US/Pacific')
+    <Arrow [2013-05-05T00:00:00-07:00]>
```

Comment:
I changed these examples because I can't think of a use case where you would want to replace the timezone for what is clearly a local time (`now`, that is).